app/assets/javascripts/components/common/ArticleViewer/containers/ArticleViewer.jsx

```diff
@@ -272,11 +273,20 @@ export class ArticleViewer extends React.Component {
         );
       }
       return (
+       this.props.article.article_id ?
```

Comment:
I suggest moving all the conditional logic into IconOpener. We really want to avoid making ArticleViewer any more complex than it already is. You can pass the article as a prop, and then let it determine its own `disabled` state.

Reply:
Ok, i can do that